pandas/_libs/src/offsets.pyx

```diff
@@ -1,367 +0,0 @@
-
-ctypedef enum time_res:
-    r_min = 0
-    r_microsecond
-    r_second
-    r_minute
-    r_hour
-    r_day
-    r_month
-    r_year
-    r_max = 98
-    r_invalid = 99
-
-
-cdef conversion_factor(time_res res1, time_res res2):
-    cdef:
-        time_res min_res, max_res
-        int64_t factor
-
-    min_res = min(res1, res2)
-    max_res = max(res1, res2)
-    factor = 1
-
-    if min_res == max_res:
-        return factor
-
-    while min_res < max_res:
-        if min_res < r_microsecond:
-            raise "Cannot convert from less than us"
-        elif min_res == r_microsecond:
-            factor *= 1000000
-            min_res = r_second
-        elif min_res == r_second:
-            factor *= 60
-            min_res = r_minute
-        elif min_res == r_minute:
-            factor *= 60
-            min_res = r_hour
-        elif min_res == r_hour:
-            factor *= 24
-            min_res = r_day
-        else:
-            raise "Cannot convert to month or year"
-
-    return factor
-
-# Logic to generate ranges
-# -----------------------------------------------------------------------------
-
-cdef inline int64_t weekend_adjustment(int64_t dow, int bkwd):
-    if dow > 4:                         # sat or sun?
-        if bkwd:                        # roll back 1 or 2 days
-            return (4 - dow)
-        else:                           # roll forward 2 or 1 days
-            return (7 - dow)
-    return 0
-
-cdef int64_t us_in_day = conversion_factor(r_microsecond, r_day)
-
-cdef class _Offset:
```

Comment:
anything useful from this file?

Reply:
I looked through it and didn't find anything worth keeping.  I've got a cython version of `tseries.offsets` in a branch that is a few PRs away from being ready to push upstream.